web/src/lib/i18n/locales/en-EN.json

```diff
@@ -0,0 +1,763 @@
+    "restore": "Restore",
+    "retry_upload": "Retry upload",
+    "reverse_geocoding_settings": "Reverse Geocoding Settings",
+    "role": "Role",
+    "scope": "Scope",
+    "signing_algorithm": "Signing algorithm",
+    "storage_label_claim": "Storage label claim",
+    "storage_label": "Storage Label",
+    "storage_quota_claim": "Storage quota claim",
+    "storage": "Storage",
+    "save": "Save",
+    "scan_new_library_files": "Scan New Library Files",
+    "scan_settings": "Scan Settings",
+    "search_albums": "Search albums",
+    "search_camera_make": "Search camera make...",
+    "search_camera_model": "Search camera model...",
+    "search_city": "Search city...",
+    "search_country": "Search country...",
+    "search_for_existing_person": "Search for existing person",
+    "search_people": "Search people",
+    "search_places": "Search places",
+    "search_state": "Search state...",
+    "search_timezone": "Search timezone...",
+    "search_your_photos": "Search your photos",
+    "search": "Search",
+    "searching_locales": "Searching locales...",
+    "select_album_cover": "Select album cover",
+    "select_all": "Select all",
+    "select_avatar_color": "Select avatar color",
+    "select_featured_photo": "Select featured photo",
+    "select_library_owner": "Select library owner",
+    "select_new_face": "Select new face",
+    "send_message": "Send message",
+    "server_stats": "Server Stats",
+    "set_slideshow_to_fullscreen": "Set Slideshow to fullscreen",
+    "set_as_album_cover": "Set as album cover",
+    "set_as_profile_picture": "Set as profile picture",
+    "set_date_of_birth": "Set date of birth",
+    "set_profile_picture": "Set profile picture",
+    "settings": "Settings",
+    "share": "Share",
+    "sharing": "Sharing",
+    "show_progress_bar": "Show Progress Bar",
+    "show_album_options": "Show album options",
+    "show_file_location": "Show file location",
+    "show_gallery": "Show gallery",
+    "show_hidden_people": "Show hidden people",
+    "show_in_timeline": "Show in timeline",
+    "show_keyboard_shortcuts": "Show keyboard shortcuts",
+    "show_person_options": "Show person options",
+    "show_search_options": "Show search options",
+    "slideshow_settings": "Slideshow settings",
+    "slideshow": "Slideshow",
+    "smart_search": "Smart Search",
+    "sort_albums_by": "Sort albums by...",
+    "stack": "Stack",
+    "state": "State",
+    "stop_motion_photo": "Stop Motion Photo",
+    "sunrise_on_the_beach": "Sunrise on the beach",
+    "swap_merge_direction": "Swap merge direction",
+    "target_resolution": "Target resolution",
+    "template": "Template",
+    "temporal_aq": "Temporal AQ",
+    "threads": "Threads",
+    "thumbnail_format": "Thumbnail format",
+    "thumbnail_resolution": "Thumbnail resolution",
+    "tone-mapping_npl": "TONE-MAPPING NPL",
+    "tone-mapping": "TONE-MAPPING",
+    "transcode_policy": "Transcode policy",
+    "two-pass_encoding": "TWO-PASS ENCODING",
```

Comment:
I think there's some inconsistency in the use of casing here (and of toUpperCase where these values are used)